docs/KVSet/README.md

```diff
@@ -0,0 +1,11 @@
+# Azure App Configuration 'appconfig/kvset' profile
+
+Azure App Configuration can be exported and imported using Azure CLI version 2.30.0 or later using the `appconfig/kvset` profile.
```

Comment:
Since you added a note, I think this sentence can be removed altogether.

Reply:
I agree, it is kind of redundant now.